content/wireless.md

```diff
@@ -16,43 +16,197 @@ section: network-troubleshooting
 tableOfContents: true
 ---
 
-WiFi issues are influenced by many different factors, including:
+If your computer can’t connect to wireless networks or the connection is unstable, use this guide to diagnose and resolve the issue.  
+Start with the quick checks, then follow the targeted diagnostic and recovery steps.
 
-- Hardware (WiFi card, access point)
-- Settings at both ends of the connection
-- The local environment
+---
+
+## Initial Troubleshooting
+
+1.Reboot the router/modem and the computer.
```

Comment:
Please put a space after the `#.` in ordered lists. This is currently being rendered as a series of regular paragraphs instead of a list since it's not valid Markdown for a list.